.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: gh-pages
+on:
+  release:
+    types: [published, edited]
```

Comment:
Do you really expect to be editing the releases where there could be some assets added/removed/updated? It is fine perhaps to edit the notes, but assets? Would not that require, another patch releases?

Reply:
Let's say you publish the release but "forget" to add the assets. Then you "edit" the release, adding the missing assets.